src/ert/_c_wrappers/enkf/config/surface_config.py

```diff
@@ -0,0 +1,18 @@
+from cwrap import BaseCClass
+
+from ert._c_wrappers import ResPrototype
+
+
+class SurfaceConfig(BaseCClass):
```

Comment:
Think this can quite be easily written as a `dataclass` instead, is already alloc`ed through python: https://github.com/equinor/ert/blob/dbffb5f2391434e1eb6e3c014b6a9856c850b4f1/src/ert/_c_wrappers/enkf/config/enkf_config_node.py#L325

Reply:
EnsemlbeConfig still holds all config_node objects in C++, thus making it a bit tricky to avoid having the C implementation of surface_config. I think we should reiterate over this later

Reply:
created issue for this: 
https://github.com/equinor/ert/issues/4516